lib/Models/CatalogItem.js

```diff
@@ -167,8 +167,18 @@ var CatalogItem = function(terria) {
      */
     this.maximumShownFeatureInfos = terria.configParameters.defaultMaximumShownFeatureInfos;
 
+    /**
+     * Options for formatting current time and timeline tic labels. See tests for examples.
+     * @type {Object}
+     */
     this.dateFormat = {};
 
+    /**
+     * Options for the value of the animation timeline at start. See tests for examples.
+     * @type {String}
+     */
+    this.timesliderInitTime = undefined;
```

Comment:
Actually, as I think about it more, I'm not sure it actually makes sense for this to be specified in `CatalogItem`.  I think this should be in the hands of the individual catalog item types.  For example, for time-dynamic WMSs, showing the latest and starting paused makes tons of sense.  For a CSV, that's almost certainly not what you want; starting at the beginning and playing is more sensible (I think, we don't have too many practical examples of this).  For CZML we don't have to decide, because the file itself specifies.